etc/templates/profile.template

```diff
@@ -11,6 +26,14 @@
 
 ##blacklist PATH
 
+# It is common to add files/dirs containing program-specific configuration
+# (often ${HOME}/PROGRAMNAME or ${HOME}/.config/PROGRAMNAME into disable-programs.inc
+# (included below) and then disable blacklisting below.
+# One way to retrieve the files a program uses is:
```

Comment:
an easier method:
- disable `shell none` and `private-bin`
- launch with --private
- join the profile with bash
- use `ls -lha` to see what was created

Reply:
Adjusted

Reply:
I prefer `tree -a`, but that is a individual decision. Because of the most program-specific dirs are hidden:`ls -lR` -> `ls -laR` (I don't think that the `-l` must be there)

Disable `shell none` is not needed, `firejail --private --ignore=private-bin --name=<NAME> --profile=<PROFILE> <PROGRAMM>` should be work.

Reply:
`tree` is not default on my Arch Linux system, `ls` is ubiquitous. will update the rest